packages/utils/src/instrument/fetch.ts

```diff
@@ -95,6 +111,55 @@ function instrumentFetch(): void {
   });
 }
 
+function instrumentFetchBodyReceived(): void {
```

Comment:
This is a relatively hefty bundle size increase 🤔 Can we not roll this into the already existing fetch fill? that would hopefully safe a bit on size there... 🤔

Reply:
I merged it into a single function but not sure if it affects bundle size..